systems/plants/viewer/drake_urdf_renderer.cpp

```diff
@@ -11,7 +12,6 @@
 #include <boost/filesystem.hpp>
 #include <boost/algorithm/string.hpp>
```

Comment:
something else to get out in a future pull.  the renderer is sufficiently distinct from the RBM classes that this doesn't have any interaction / constraints on the interactions w/ matlab.